feat: add about author and real world entities list to README

Kirill-Stulov opened the pull request:
#comment-starting
- feat: add about author static page
- docs: add list of real-world entities to README

Kirill-Stulov committed:
Moved a2f95be by cherry-pick: done: ru locale, app version, about app

Kirill-Stulov committed:
feat: add about author and real world entities list to README
#comment-starting
- feat: add about author static page
- docs: add list of real-world entities to README

VidgarVii commented on README.md:
```diff
@@ -1,48 +1,71 @@
+  -Categories
+    Tests can be organized into categories/groups
+  
+  -Some other entities that may make sense:
```
Этого всего не будет. Но надо куда-то будет записывать результат прохождения теста

Kirill-Stulov replied:
Удалил категории тестов (- Categories), оставил результаты тестов (-Test Results).

VidgarVii commented on public/about/author.html:
```diff
@@ -0,0 +1,38 @@
+<h4>About author</h4>
```
файлы из `public` отдаются минуя контроллеры "как есть", т.е. без лэйаута
это означает, что здесь должен быть полноценная `html` страница, которая открывается тегом `<!DOCTYPE html>` и содержит теги `<html>`, `<head>`, `<body>` и т.д.

Kirill-Stulov replied:
Сделал страницы в public полноценными html страницами со стандартным набором тегов.

Kirill-Stulov committed:
feat: updated about author and real world entities list in README
#comment-starting
- feat: updated about author and about project static pages
- docs: removed categories item of real-world entities in README

Kirill-Stulov reviewed (commented):
Свежий коммит, после обновлений с учетом комментариев.

Kirill-Stulov committed:
feat: fixed links in public static pages, updated real world entities list in README
#comment-starting
- feat: fixed links by: _blank; nofollow and noopener in  about author and about project static pages
- docs: updated list of real-world entities in README

VidgarVii commented on README.md:
```diff
@@ -1,48 +1,63 @@
-
+    - Публикация результатов тестов в соцсети
+
+List of real world entities:
```
Будут еще Категории (для тестов)

Kirill-Stulov replied:
Вернул, до этого неверно понял и убрал не то.

VidgarVii commented on public/about/about_project.html:
```diff
@@ -1,59 +1,54 @@
+<!DOCTYPE html>
```
по ТЗ страница должна открываться по пути /about

Kirill-Stulov replied:
Перенес

VidgarVii commented on public/about/about_project.html:
```diff
@@ -1,59 +1,54 @@
+      <section>
+
+        <p>
+          Курс проекта: <a href="http://www.thinknetica.com">http://www.thinknetica.com</a>
```
ссылки на сторонние ресурсы правильно открывать в новой вкладке и хорошо бы добавить опцию rel=nofollow rel=noopener. Так клиента сохраним и поискового робота дальше не пустим

Kirill-Stulov replied:
Обновил

VidgarVii commented on public/1_all_static_pages_here.txt:
```diff
@@ -1 +1,3 @@
 all static pages, which contains public info, are here
```
этот файл лишний

Kirill-Stulov replied:
Убрал

Kirill-Stulov committed:
feat: locale; location; timezone; app version updated so updates are seen in Pull Request

	- feat: updated application.rb
	- feat: updated version.rb
	- feat: updated ru.yml

VidgarVii commented on config/initializers/version.rb:
```diff
@@ -1 +1,2 @@
+# !!! done - app version
 APP_VERSION = '0.0.1'
```
1. Раное еще для патча, когда нет еще минорной версии
2. Не стоит засорять глобальное пространство
версия относится к приложению, так что и константу нужно объявить в его контексте

посмотри к примеру в файл `config/application.rb` в какой модуль обернуто все твое приложение (9-я строка)

```ruby
module TestGuru
  class Application < Rails::Application
    VERSION = '0.1.0'.freeze
  end
end
```

VidgarVii commented on config/application.rb:
```diff
@@ -21,6 +21,7 @@ class Application < Rails::Application
+    # !!! done timezone and switched to ru default locale
     # config.time_zone = "Central Time (US & Canada)"
     config.time_zone = 'Asia/Tashkent'
     config.active_record.default_timezone = :local
```
так делать не рекомендуется (`default_timezone = :local`)

стандарт - хранение времени в UTC
это намного удобнее для приложений, в которых используется несколько таймзон (приложение всегда может стать таким)

VidgarVii replied:
нет конфига локали по умолчанию

Kirill-Stulov committed:
feat: fixed default locale; timezone to UTC; app version module

	- feat: updated application.rb
	- feat: updated version.rb

Kirill-Stulov committed:
feat: fixed default timezone to UTC
